ui/ember-cli-build.js

```diff
@@ -34,8 +34,9 @@ const appConfig = {
       return `${config.rootURL.replace(/\/$/, '')}${filePath}`;
     },
   },
-  babel: {
-    plugins: [['inline-json-import', {}]],
```

Comment:
I need to double-check this still works, but webpack should let you do this via loaders.